internal/provider/provider.go

```diff
@@ -65,6 +65,8 @@ func New() *schema.Provider {
 						transition = "start"
 					}
 					rd.Set("transition", transition)
+					rd.Set("username", os.Getenv("CODER_WORKSPACE_USERNAME"))
```

Comment:
thoughts on `owner` or `CODER_WORKSPACE_OWNER` instead? to me, username sounds like a property unique to the workspace.